workshop_1.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Welcome to IndieAGI workshop 1!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## By the end of this workshop you will be able to:\n",
+    "* Excecute python code in a jupyter notebook\n",
+    "* Know the importance of each section of code\n",
+    "* Feel more enabled than you were yesterday"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Instructions to run code\n",
+    "<img src=\"./images/run_cell_instructions.png\" style=\"width: 50%; height: auto;\">\n",
+    "\n",
+    "1. Click on the code block you want to run. \n",
+    "2. Press the play button in the toolbar above to exceute the code.  \n",
+    "  \n",
+    "(You can also press ```shift+enter``` to run the code block and move to the next one.)  \n",
+    "\n",
+    "**Task**: Try in on the next cell"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install\n",
+    "%pip install langchain huggingface_hub\n",
+    "\n",
+    "\n",
+    "# Do you have any questions?\n",
+    "# Any feedback if something is making sense or not? \n",
+    "# Type below following the \"#\" symbol:\n",
+    "#\n",
+    "#\n",
+    "#"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Obtaining Huggingface access token\n",
+    "1. Go to https://huggingface.co/settings/token\n",
+    "2. Create and login to your account\n",
+    "3. Your screen should look something like this at this point:  \n",
+    "   <img src=\"./images/new_access_token_page.png\" style=\"width: 50%; height: auto;\">  \n",
```

Comment:
On github repo the images don't render on a new line unlike on VScode. I am assuming it will render correctly on the jupyter hub but I am yet to test.

Reply:
I believe you need to use a double space and an actual newline instead of an escape newline.